charts/intel-gpu-resource-driver/templates/resource-driver.yaml

```diff
@@ -70,11 +39,6 @@ spec:
           mountPath: /etc/cdi
         - name: varruncdi
           mountPath: /var/run/cdi
-        - name: defaults
-          mountPath: "/defaults"
-          readOnly: true
-        - name: sysfs
-          mountPath: "/sysfs"
```

Comment:
the sysfs itself has to stay, the driver won't work without it, only faking sysfs needed to be removed. The env var for it, and volumeMount are also needed.